.gitignore

```diff
@@ -10,7 +10,6 @@ gutenberg.zip
 *.log
 phpcs.xml
 yarn.lock
-docker-compose.override.yml
```

Comment:
Probably worth keeping this around:

* it's benign
* once a user pulls these commits, their override file will still be in place so an accidental commit is bordering on likely.

Reply:
I'm inclined to remove it, as while it should be benign, it'd be the source of the weirdest bugs in existence if it turns out that it does clash in some way. It's better to ensure the file is deleted, rather than let it hang around.

Even if an accidental commit happens, I don't think a PR with the accidental commit in it would be merged.